script/test

```diff
@@ -41,40 +41,19 @@ readonly COLOR_PASS='\033[0;32m'
 readonly COLOR_FAIL='\033[0;31m'
 readonly COLOR_NONE='\033[0m'
 
-print_result() {
-    pass=$1
-
-    if ${pass}; then
-        prefix="${COLOR_PASS}PASS${COLOR_NONE}"
-    else
-        prefix="${COLOR_FAIL}FAIL${COLOR_NONE}"
-    fi
-    shift
-
-    echo -e "${prefix}" $*
-
-    ${pass}
-}
-
-do_build() {
-    [[ -d "${BUILD_DIR}" ]] || mkdir "${BUILD_DIR}"
-    (cd "${BUILD_DIR}" && cmake .. -DCMAKE_TOOLCHAIN_FILE=../cmake/arm-none-eabi.cmake -DPLATFORM_NAME="${PLATFORM_NAME}" && make -j${BUILD_JOBS})
-}
-
 do_clean() {
     [[ ! -d "${BUILD_DIR}" ]] || rm -rvf "${BUILD_DIR}"
+}
 
-    do_build
+do_build() {
+    do_clean
+    mkdir "${BUILD_DIR}"
+    (cd "${BUILD_DIR}" && cmake .. $@ && make -j${BUILD_JOBS})
 }
 
 check_pretty() {
-    pass=false
-    find src include \( -name '*.c' -o -name '*.cpp' -o -name '*.h' -o -name '*.hpp' \) | while read file; do
-        echo Checking "${file}"
-        third_party/openthread/repo/script/clang-format-check.sh "${file}" || break
-        pass=true
-    done
-    print_result "${pass}" pretty check
+    find src include third_party/*/port \( -name '*.c' -o -name '*.cpp' -o -name '*.h' -o -name '*.hpp' \) |\
```

Comment:
Will this give us which file is failing?

Reply:
The diff output will tell us about the filename.